scripts/experiment_1_same_objective_diff_batches/nbconverted/5_similarity_analysis.py

```diff
@@ -124,10 +168,11 @@
 svcca_umap_df
```

Comment:
The reason for the results for all being 99% are because of a typo.  I was accidentally comparing svcca(input, input) instead of svcca(input, input + batch effects)